### DISCUSSION TITLE
任务函数在哪呐

### DISCUSSION THREAD

**Discussion by lVRuixiao** [peng-zhihui/ElectronBot#49] [Q&A]
兄弟们有没有找到各个任务的代码在哪里呐，我在CubeMX里面的FreeRTOS只看到了一个初始任务，进工程里也只找到了这个任务函数
void StartDefaultTask(void *argument)，而且里面就一个USB的初始化哇。。但是电机 、显示屏这些任务都没看见也。

**Comment by zhugexunyu** [comment]
main.cpp里有

**Comment by lVRuixiao** [comment]
main里没有吧。。任务就只找了这一个，main里就初始化和开始调度，大循环里也是空的也。（纯小白，好哥哥别骂我）

**Comment by peng-zhihui** [comment]
main.cpp，不是main.c，初始化完成之后就到C++环境了

**Reply by ImYSY-22** [reply]
虽然看不懂，但大为震撼

**Comment by lVRuixiao** [comment]
嗷嗷，我强行用的KEIL看的源码，没GO TO到Main，现在找到咯，谢谢稚晖哥，偶像！

**Comment by MrJfq** [comment]
main.cpp在哪里？没找到

**Reply by coolflyreg** [reply]
UserApp文件夹
